GameServer/Logic/Maps/SummonersRift.cs

```diff
@@ -141,26 +142,30 @@ class SummonersRift : Map
 
         private static readonly Dictionary<TurretType, int[]> _turretItems = new Dictionary<TurretType, int[]>
         {
-            { TurretType.OuterTurret, new int[] { 1500, 1501, 1502, 1503 } },
-            { TurretType.InnerTurret, new int[] { 1500, 1501, 1502, 1503, 1504 } },
-            { TurretType.InhibitorTurret, new int[] { 1501, 1502, 1503, 1505 } },
-            { TurretType.NexusTurret, new int[] { 1501, 1502, 1503, 1505 } }
+            { TurretType.OuterTurret, new[] { 1500, 1501, 1502, 1503 } },
+            { TurretType.InnerTurret, new[] { 1500, 1501, 1502, 1503, 1504 } },
+            { TurretType.InhibitorTurret, new[] { 1501, 1502, 1503, 1505 } },
+            { TurretType.NexusTurret, new[] { 1501, 1502, 1503, 1505 } }
         };
 
-        private RAFManager _rafManager = Program.ResolveDependency<RAFManager>();
         private Logger _logger = Program.ResolveDependency<Logger>();
-        private NetworkIdManager _networkIdManager = Program.ResolveDependency<NetworkIdManager>();
-        private int _cannonMinionCount = 0;
+        private int _cannonMinionCount;
 
         public SummonersRift(Game game) : base(game, 15 * 1000, 30 * 1000, 90 * 1000, true, 1)
         {
-            RAF.AIMesh aiMesh = null;
-            if (!_rafManager.readAIMesh("LEVELS/Map1/AIPath.aimesh", out aiMesh))
+            var path = System.IO.Path.Combine(Program.ExecutingDirectory, "Content", "Data",
```

Comment:
When expanding arguments, please expand them in some consistent/easy to read format (one argument = one line, for example)
```CS
var path = System.IO.Path.Combine(
    Program.ExecutingDirectory,
    "Content",
    "Data",
    _game.Config.ContentManager.GameModeName,
    "AIMesh",
    "Map" + GetMapId(),
    "AIPath.aimesh"
);
```